pkg/version/plugin2/commonSpecForVCH.go

```diff
@@ -0,0 +1,127 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package plugin2
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/vmware/vic/lib/migration/manager"
```

Comment:
In general, packages in `pkg` shouldn't be referencing packages in `lib` - pkg is for supporting code that isn't necessarily VIC specific.

I'm also unsure about the naming `plugin2`, and `plugin3` - this implies an ordering of some kind and doesn't provide any information about what/why they exist.